Sources/ZcashLightClientKit/Synchronizer.swift

```diff
@@ -144,39 +144,39 @@ public protocol Synchronizer: AnyObject {
     func stop()
 
     /// Gets the sapling shielded address for the given account.
-    /// - Parameter accountIndex: the optional accountId whose address is of interest. By default, the first account is used.
+    /// - Parameter account: the account whose address is of interest. By default, the first account is used.
     /// - Returns the address or nil if account index is incorrect
-    func getSaplingAddress(accountIndex: Int) async throws -> SaplingAddress
+    func getSaplingAddress(account: Zip32Account) async throws -> SaplingAddress
 
     /// Gets the unified address for the given account.
-    /// - Parameter accountIndex: the optional accountId whose address is of interest. By default, the first account is used.
+    /// - Parameter account: the account whose address is of interest. By default, the first account is used.
     /// - Returns the address or nil if account index is incorrect
-    func getUnifiedAddress(accountIndex: Int) async throws -> UnifiedAddress
+    func getUnifiedAddress(account: Zip32Account) async throws -> UnifiedAddress
 
     /// Gets the transparent address for the given account.
-    /// - Parameter accountIndex: the optional accountId whose address is of interest. By default, the first account is used.
+    /// - Parameter account: the account whose address is of interest. By default, the first account is used.
```

Comment:
```suggestion
    /// - Parameter accountIndex: the ZIP 32 index of the account whose address is of interest.
```
There is no default for this parameter.

Reply:
Updated